src/main/java/com/jm/lms/studentms/model/EditCourse.java

```diff
@@ -0,0 +1,82 @@
+package com.jm.lms.studentms.model;
+
+import org.hibernate.annotations.GenericGenerator;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+
+@Entity
+@Table(name = "EDITCOURSE")
```

Comment:
Change table name to course_details

Reply:
Changed the table name to course_details